webrtc.html

```diff
@@ -30,9 +30,10 @@
 
     <p>To go into Proposed Recommendation status, the group expects to demonstrate implementation of each feature in at least two deployed browsers, and at least one implementation of each optional feature. Mandatory feature with only one implementation may be marked as optional in a revised Candidate Recommendation where applicable.</p>
 
-    <p>The following features are marked as risk:</p>
+    <p>The following features are marked as at risk:</p>
     <ul>
       <li>The value <a href="#dom-rtcrtcpmuxpolicy-negotiate"><code>negotiate</code></a> of RTCRtcpMuxPolicy</li>
+      <li>The <code>rid</code> attribute of <code><a>RTCRtpCodingParameters</a></code></li>
```

Comment:
It's just its presence in RTCRtpDecodingParameters that's at risk; it would still go in RTCRtpEncodingParameters

Reply:
The two pieces of this edit are inconsistent; one says that simulcast reception is the feature at risk, the other says that the rid field (which only makes sense as an indication of simulcast reception) is the feature at risk.
I think the IDL feature at risk should be the "encodings" parameter of the RTCRTPDecodingParameters dictionary, not the "rid" parameter - on the encoding side, it's not at risk.